### PR TITLE
Updated Readme of the FAIRifier

### PR DESCRIPTION
**Pull request by Shamanou**
I changed the readme to contain text and links about dtl and related project putting the FAIRifier in context

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/d122f3edbb6e4ebfa8c74c4b219ce3a2)](https://www.codacy.com/app/Shamanou/FAIRifier?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Shamanou/FAIRifier&amp;utm_campaign=Badge_Grade)
 
+The FAIRifier is a tool to make messy data [FAIR](https://www.dtls.nl/fair-data/).
+
+The FAIRifier is a tool to make data FAIR(Findable, Acessable, Interoperable and Reusable).
```
Comment (kburger) [inline README.md]: Repetition of the first line. This should be rewritten to explain more about the tool: where it came from, what it is based on, what changes were made to the original platform, etc.

Comment (kburger) [inline README.md]: Might be good to link out to the principles as well, not everyone might click on the first link.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/d122f3edbb6e4ebfa8c74c4b219ce3a2)](https://www.codacy.com/app/Shamanou/FAIRifier?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Shamanou/FAIRifier&amp;utm_campaign=Badge_Grade)
 
+The FAIRifier is a tool to make messy data [FAIR](https://www.dtls.nl/fair-data/).
+
+The FAIRifier is a tool to make data FAIR(Findable, Acessable, Interoperable and Reusable). 
+The FAIRifier is developed at [DTL](https://www.dtls.nl/)(the Dutch Tech Centre for life sciences) and should be used if a user wants
```
Comment (kburger) [inline README.md]: Could be part of another section, which explains the use case for DTL. The "should be used if a user wants..." could go to a separate section on usage examples.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+to make non-FAIR datasets FAIR and push the metadata to a [FDP](https://github.com/DTL-FAIRData/FAIRDataPoint)(Fair Data Point).
```
Comment (kburger) [inline README.md]: If the FDP is mentioned, it might need a better introduction. We can discuss a restructure of the README section if you want.

**Review by kburger — CHANGES_REQUESTED** [review]
There is too little information for new users who want to get a quick feel for the project. You can check out existing OSS projects to get an impression of how they introduce their tools, for example:

https://github.com/OpenRefine/OpenRefine
https://github.com/google/guava
https://github.com/spring-projects/spring-framework

**Review by rajaram5 — CHANGES_REQUESTED** [review]
Hi Shamanou,  I assume you moved some of the content from (master branch) README.md file to wiki. It is also a good idea to provide pointers to that content in  (development branch) README.md file. Maybe make a how to use section in README.md file

**Comment by kburger** [comment]
For a next iteration, you should think about adding section headers. For now, if you can make the changes Rajaram suggested, I'm OK with this.

File: README.md [hunk 4]
```diff
@@ -32,3 +52,5 @@ Run the ./refine file
 ```
 ./refine
 ```
+
+You can find more information on the [FAIRifier wiki](https://github.com/DTL-FAIRData/FAIRifier/wiki).
```
Comment (Shamanou) [inline README.md]: This links to the wiki

Comment (kburger) [reply README.md]: I think Rajaram meant to add specific links to specific pages on the wiki. Makes more sense to me as well. Maybe you can update the main wiki page as well, right now it looks like it's an older version of the readme?

Comment (Shamanou) [reply README.md]: no its an old version of the readme and also some other pages. Should i add information to the wiki main page?

Comment (Shamanou) [reply README.md]: i could update it with the information in the readme?

**Comment by kburger** [comment]
We found a good template that you(/we all) can use as a base for a good readme: [A template to make good README.md](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2) (and read [this one](https://github.com/zalando/zalando-howto-open-source/blob/master/READMEtemplate.md) for a more in depth explanation on certain sections)

I propose to use this template, with the following exceptions:
- leave out the 'Contributing' section for now
- leave out the 'Authors' section

As for the wiki, take a look at these open source projects for inspiration on how they organised their wiki pages. Generally, they have a main page with an explanation and links to the wiki sections.
- https://github.com/spring-projects/spring-framework/wiki
- https://github.com/Netflix/Hystrix/wiki
- https://github.com/mcMMO-Dev/mcMMO/wiki

Try to update the readme and wiki pages as much as you can according to these examples.

**Review by rajaram5 — APPROVED** [review]

**Comment by rajaram5** [comment]
For now I will approve it but we have to revisit this issue in later sprint

**Comment by kburger** [comment]
I'm with Rajaram, and be careful in the future with pushing to an open PR. This PR now incorporates [f41724c](https://github.com/DTL-FAIRData/FAIRifier/pull/29/commits/f41724c1e3ecd6ede456a9fc72f236689f02422c) and [ce08307](https://github.com/DTL-FAIRData/FAIRifier/pull/29/commits/ce083078f7a518118a3596e66c9c93a5f48732ff) as well, which I didn't properly review.

**Review by kburger — APPROVED** [review]

**Pull request merged**